src/base/object_representation_sr.c

```diff
@@ -4690,3 +4714,39 @@ or_mvcc_get_prev_version_lsa (OR_BUF * buf, int mvcc_flags, LOG_LSA * prev_versi
 
   return NO_ERROR;
 }
+
+/* 
+ * or_is_replication_key_candidate() -
+ *   return : true if the index is a replication-key candidate
+ *            (PRIMARY KEY, or UNIQUE with all key attributes NOT NULL)
+ *   index(in): OR_INDEX pointer (may be NULL)
+ */
+bool
+or_is_replication_key_candidate (const OR_INDEX * index)
```

Comment:
후보키(candidate key)라는 표현이 더 많이 사용되고, 적합하다고 생각합니다.
```suggestion
or_is_replication_candidate_key (const OR_INDEX * index)
```

Reply:
@hornetmj 반영했습니다.